admin/metrics/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+
+class OSFStatistic(models.Model):
```

Comment:
On a historical note, the Admin was originally outside of the OSF repo, but essentially had the entire OSF as a requirement, tied to a specific build. So when I took over I decided it would be better to emulate the API rather than importing the entire OSF. In the future it may be possible to extract the admin and let it stand alone, but that assumes a huge overhaul of the OSF.

Although it may seem idiosyncratic, really #5001 uses a similar naming scheme to separate logs from being normal Django admin logs to being OSF related.

Changed to `OSFWebsiteStatistics`